README.md

```diff
@@ -1 +1,583 @@
+- Invoke the Travel Agency service by sending a POST request to arrange a tour.
+
+```bash
+   curl -v -X POST -d \
+   '{"ArrivalDate":"2007-11-06", "DepartureDate":"2007-11-06", "From":"CMB", "To":"DXB", 
+   "VehicleType":"Car", "Location":"Changi"}' \
+   "http://localhost:9090/travel/arrangeTour" -H "Content-Type:application/json" 
+```
+
+- Travel Agency service will send a response similar to the following:
+    
+```bash
+   HTTP/1.1 200 OK
+    {
+        "Flight":{
+            "flightNo":1,"airline":"Emirates","arrivalDate":"2007-11-06+05:30",
+            "departureDate":"2007-11-06+05:30","to":"DXB","rom":"CMB","price":100
+        },
+        "Hotel":{
+            "HotelName":"Elizabeth","FromDate":"2007-11-06","ToDate":"2007-11-06","DistanceToLocation":2
+        },
+        "Vehicle":{
+            "company":"Sixt","arrivalDate":"2007-11-06+05:30","departureDate":"2007-11-06+05:30",
+            "vehicleType":"Car","price":30
+        }
+    }
+```
+## Observability 
+Ballerina is by default observable. Meaning, you can easily observe your services, resources, etc. Observability configurations are defined in the `ballerina.conf` file. A sample configuration file can be found in `microservices-observability/guide/` directory.
+```ballerina
+[b7a.log]
+level="DEBUG"
+```
+Define the log level for the Ballerina services.
+
+To start the Ballerina services using the configuration file, run the following command
+```
+   $ ballerina run --config travel_agency/ballerina.conf <package_name>
+```
+When you execute the ```ballerina run``` command from the ```microservices-observability/guide/``` location (where the `ballerina.conf` file is), the configuration file will get applied to every service that you are starting. Therefore, you don't need to pass the `--config` argument explicitly.
+
+### Tracing 
+You can monitor Ballerina services using in-built tracing capabilities of Ballerina. We'll use [Jaeger](https://github.com/jaegertracing/jaeger) as the distributed tracing system.
+Follow the below steps to use tracing with Ballerina.
+- Add below configurations to the `ballerina.conf` file
+```
+[b7a.observability.tracing]
+# Flag to enable Tracing
+enabled=true
+
+[b7a.observability.tracing.jaeger]
+# Jaeger Host and Port, to which the tracing information will be pushed by Ballerina
+reporter.hostname="localhost"
+reporter.port=5775
+sampler.param=1.0
+sampler.type="const"
+reporter.flush.interval.ms=2000
+reporter.log.spans=true
+reporter.max.buffer.spans=1000
+```
+
+- Run Jaeger Docker image using the following command. (If you haven't already started Jaeger as instructed in the [prerequisites](#prerequisites) section)
+```bash
+   $ docker run -d -p5775:5775/udp -p6831:6831/udp -p6832:6832/udp -p5778:5778 \
+   -p16686:16686 p14268:14268 jaegertracing/all-in-one:latest
+```
+
+- Observe the tracers using Jaeger UI using following URL after doing some service invocations on Travel Agency service.
```

Comment:
Invoke the Travel Agency service several times and observe the tracers in the Jaeger UI using the following URL